foundation/aissemble-foundation-core-python/src/policy_manager/policy/policy.py

```diff
@@ -83,17 +84,12 @@ def set_deprecated_targetConfigurations(self, new_value: ConfiguredTarget):
     # Links ConfiguredRule 'targetConfigurations' attribute to 'set_deprecated_targetConfigurations()' method to support
     # people still assigning values to the old attribute.
     def __setattr__(self, key, val):
-        method = self.__config__.property_set_methods.get(key)
+        method = self.__deprecated_set_methods.get(key)
```

Comment:
Q: what's actually happening here? I'm not very familiar with pylint and it's not obvious to me what this change does, functionally.

Reply:
We're deprecating the `target` attribute on a policy to instead be a list of targets. People still using the old attribute like this:
```
MyPolicy.target = MyTarget
```
will get a nice warning message. This just ties our custom function to the Python setter.

Reply:
`property_set_methods` did not exist on the pydantic config class, so pylint through an error. We're moving the attribute to just live on our local class instead.